feat: new package for udir-symbols

adrianflatner opened the pull request.

hannehaugland commented on @udir-design/symbols/README.md:
````diff
@@ -0,0 +1,72 @@
+## Installere biblioteket
+
+```bash
+npm add @udir-design/symbols@beta
````
Hvorfor rett ut i Beta?

hannehaugland commented on @udir-design/symbols/README.md:
````diff
@@ -0,0 +1,72 @@
+}
+```
+
+Tips: Bruk `title` prop for tilgjengelige symboler.
````
Hadde nettopp en runde på title vs aria-label osv ifm. ikoner. Samsnakke hva Une landa på der så vi anbefaler samme, med mindre de burde brukes ulikt?

hannehaugland reviewed (commented):
Kult! Lurer på litt ting:)

adrianflatner commented on @udir-design/symbols/eslint.config.js:
```diff
@@ -0,0 +1,7 @@
+import baseConfig from '../../eslint.config.js';
+
+export default defineConfig(nxEslintPlugin.configs['flat/react'], baseConfig, {
+  ignores: ['src/*.tsx'],
```
Litt for streng eslint. Det fjernes en tom linje som ikke har noe å si. Mulig det kan løses på en annen måte!

adrianflatner committed:
fix(icons): new id for svg icons

BREAKING CHANGE: `<svg><use href="path/to/icon.svg#root"></svg>`
must be changed to `<svg><use href="path/to/icon.svg#icon"></svg>`.
This does not affect usage of icons as React components.

adrianflatner committed:
feat: new package for udir-symbols

unekinn commented on @udir-design/react/src/symbols/Symboler.mdx:
```diff
@@ -0,0 +1,25 @@
+
+# Symboler
+
+Symboler gir visuell støtte til å oppfatte et innhold og bør stå sammen med tekst. De er tilgjengelig både som React-komponenter og som svg-filer. Vi anbefaler å bruke React-komponentene når du bruker React. Trykk på en av symbolene i galleriet under for å se flere varianter og hvordan de brukes i kode.
```
```suggestion
Symboler gir visuell støtte til å oppfatte et innhold og bør stå sammen med tekst. De er tilgjengelig både som React-komponenter og som svg-filer. Vi anbefaler å bruke React-komponentene når du bruker React. Trykk på ett av symbolene i galleriet under for å se flere varianter og hvordan de brukes i kode.
```

unekinn commented on @udir-design/symbols/config/figma/fetch-symbols.ts:
```diff
@@ -0,0 +1,114 @@
+        {
+          headers: {
+            'Content-type': 'application/json',
+            'X-FIGMA-TOKEN': process.env.FIGMA_TOKEN ?? '',
```
Kan du legge inn denne som kommentar (utan verdi) i `.env`? Sånn:
```
#FIGMA_TOKEN=
```

unekinn reviewed (changes requested):
Veldig bra jobba! Litt pirk fra meg.

unekinn committed:
build: ensure temporary files don't mess up eslint

unekinn committed:
build: try to guard against a possible temp-file generated by svgr crashing eslint???

This file does not exist, so it's weird that eslint sometimes complains that it doesn't

ENOENT: no such file or directory, open '/home/runner/work/designsystem/designsystem/@udir-design/symbols/.svgrrc.mjs'
